### ISSUE TITLE
please authorize my bot

### ISSUE THREAD

**Issue by AshleyJ24** [EdgePlus-MIA/EdgyBot#293]
Hello
     I paid $45 through cashapp for this and then you went offline.  I only was able to use it for maybe 10 minutes.  Now you won't allow sign in.  Please authorize my account so I can get my 2 classes finished.  Thank you for your time. 
My Cashapp is $konacash24

**Comment by AshleyJ24** [comment]
It says incorrect information, account does not exist.  I am using correct username and password that I set up with you guys.   Just please PLEASE fix this.

**Comment by EdgePlus-MIA** [comment]
No, you are using incorrect initials for ur account. Enter your Edgenuity initials, the same ones on the top right corner for Edgenuity when making an order.
